scrabble.js

```diff
@@ -0,0 +1,34 @@
+class Scrabble {

```

Comment:
Move all of the code in this file into the file **src/scrabble.js** so that you can test your code using `npm test` after running `npm install`